Utilities/Pipelines/libHttpClient.CI.yml

```diff
@@ -0,0 +1,168 @@
+# Refs:
+# - https://docs.microsoft.com/en-us/azure/devops/pipelines/process/scheduled-triggers
+# - https://docs.microsoft.com/en-us/azure/devops/pipelines/process/scheduled-triggers#cron-syntax
+schedules:
+  # Schedule is specified in `cron` syntax. The below means: "8am UTC (midnight PST) every day".
+  - cron: "0 8 * * *"
+    displayName: Daily build
+    branches:
+      include:
+        - main
+    always: true
+
+# Name builds with the definition name, date, and build-count-for-that-day. For
+# example, "libHttpClient.CI_20210214.1".
+# Refs:
+# - https://docs.microsoft.com/en-us/azure/devops/pipelines/process/run-number
+# - https://docs.microsoft.com/en-us/azure/devops/pipelines/build/variables
+name: $(Build.DefinitionName)_$(date:yyMM).$(date:dd)$(rev:rrr)
+
+jobs:
+
+  ####################
+  # Visual Studio 2017
+  ####################
+
+  - job: Win32Vs17Build
+    displayName: libHttpClient Win32/UWP VS2017 Build
+    pool:
+      name: xbl-1es-vs2017
+    timeoutInMinutes: 180
+    strategy:
+      matrix:
+        x86_Debug:
+          Platform: x86
+          Configuration: Debug
+        x64_Debug:
+          Platform: x64
+          Configuration: Debug
+        x86_Release:
+          Platform: x86
+          Configuration: Release
+        x64_Release:
+          Platform: x64
+          Configuration: Release
+    steps:
+      - template: Tasks/vs2017-build.yml
+        parameters:
+          platform: $(Platform)
+          configuration: $(Configuration)
+
+  ####################
+  # Visual Studio 2019
+  ####################
+
+  - job: Win32Vs19Build
+    displayName: libHttpClient Win32/UWP VS2019 Build
+    pool:
+      name: xbl-1es-vs2019
+    timeoutInMinutes: 180
+    strategy:
+      matrix:
+        x86_Debug:
+          Platform: x86
+          Configuration: Debug
+        x64_Debug:
+          Platform: x64
+          Configuration: Debug
+        x86_Release:
+          Platform: x86
+          Configuration: Release
+        x64_Release:
+          Platform: x64
+          Configuration: Release
+    steps:
+      - template: Tasks/vs2019-build.yml
+        parameters:
+          platform: $(Platform)
+          configuration: $(Configuration)
+
+  ####################
+  # Android
+  ####################
+
+  - job: AndroidBuild
+    displayName: libHttpClient Android Build
+    pool:
+      vmImage: internal-macos-10.15
+    timeoutInMinutes: 180
+    strategy:
+      matrix:
+        x64_Release:
+          Platform: x64
+          Configuration: Release
+        x64_Debug:
+          Platform: x64
+          Configuration: Debug
+    steps:
+      - template: Tasks/android-build.yml
+        parameters:
+          platform: $(Platform)
+          configuration: $(Configuration)
+    
+  ####################
+  # iOS
+  ####################
+
+  - job: iOSBuild
+    displayName: libHttpClient iOS Build
+    pool:
+      vmImage: internal-macos-10.15
```

Comment:
Do we need the internal pool specifically?

Reply:
No idea. I'm just mirroring existing pipeline so I'm not going to mess with it since I need to move on to other tasks but feel free to improve post this PR if you like